client/src/common/ErrorSnackbar.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import {
+  Typography,
+  Grid,
+  Snackbar,
+  SnackbarContent,
+  makeStyles
+} from "@material-ui/core";
+import { Error } from "@material-ui/icons";
+
+//CSS styles
+const useStyles = makeStyles({
+  snackbarContentError: {
+    backgroundColor: "#cc3300",
+    height: "5rem"
+  },
+  icon: {
+    marginRight: "10px"
+  }
+});
+
+const ErrorSnackbar = ({ open, onClose, message }) => {
```

Comment:
There is a lot of repeated code between the Success and Error Snackbar. Alternatively, I would love to see only one general type of snackbar, and you can pass in the type, and based on the type, you render a different background

Reply:
Updated. Combined success and error snackbar. You can pass in type param to render different snackbar (error or success)